app/views/spree/admin/adjustments/_adjustments_table.html.haml

```diff
@@ -19,5 +19,6 @@
         %td.align-center.included-tax= adjustment.display_included_tax.to_html
         - unless @order.canceled?
           %td.actions
-            = link_to_edit adjustment, no_text: true
-            = link_to_delete adjustment, no_text: true
+            - if adjustment.originator_type.nil?
+              = link_to_edit adjustment, no_text: true
+              = link_to_delete adjustment, no_text: true
```

Comment:
There are already some issues around this in master where the current logic doesn't always work well or produces unexpected results. If the user manually edits the amount on a shipping or payment fee adjustment, the value will be reset when they hit "save and recalculate fees", as those values are calculated based on the related shipping or payment methods.

So here we're removing those options to avoid confusion. So in that scenario you outlined, the hub manager could change the shipping method to one with a different cost. Otherwise yes, manual adjustments for negative or positive values can be added at any time, so if the hub manager wanted to refund $5 on shipping because a friend picked up the order that day, they could just add an adjustment for `-$5`, and label it `"Partial shipping refund for pickup"` (or whatever they like).

How does that sound?

Reply:
Yes, sounds much cleaner and clearer for everyone.